VTEX - Pick and Pack API.json

```diff
@@ -0,0 +1,4196 @@
+                "type": "object",
+                "properties": {
+                    "message": {
+                        "type": "string"
+                    },
+                    "code": {
+                        "type": "string"
+                    },
+                    "reason": {
+                        "type": "string"
+                    },
+                    "requestId": {
+                        "type": "string"
+                    }
+                },
+                "example": {
+                    "message": "User is not authorized to access this resource with an explicit deny",
+                    "code": "EXPIRED_TOKEN",
+                    "reason": "Expired token",
+                    "requestId": "1a36389e-0050-436e-a1d0-14cfb6508c2f"
+                }
+            },
+            "403Forbidden1": {
+                "required": [
+                    "type",
+                    "code",
+                    "message",
+                    "date",
+                    "requestId"
+                ],
+                "type": "object",
+                "properties": {
+                    "type": {
+                        "type": "string",
+                        "description": "Type of the error."
+                    },
+                    "code": {
+                        "type": "string",
+                        "description": "Error code."
+                    },
+                    "message": {
+                        "type": "string",
+                        "description": "Error message."
+                    },
+                    "date": {
+                        "type": "string",
+                        "description": "Date and time when the error occurred."
+                    },
+                    "requestId": {
+                        "type": "string",
+                        "description": "Request ID."
+                    }
+                },
+                "example": {
+                    "type": "business",
+                    "code": "SHR0007B",
+                    "message": "You do not have access to this order [1481120515528-01]",
+                    "date": "2024-12-04T01:03:29.495Z",
+                    "requestId": "3d292319-2129-47b6-a748-92a8be02576a"
+                }
+            },
+            "SetOrderDelivery": {
+                "required": [
+                    "deliveredDate",
+                    "events",
+                    "orderId"
+                ],
+                "type": "object",
+                "properties": {
+                    "deliveredDate": {
+                        "type": "string",
+                        "format": "date-time",
+                        "description": "Date and time when the order was delivered."
+                    },
+                    "events": {
+                        "type": "array",
+                        "description": "List of events that occurred during the delivery process.",

```

Comment:
I am not sure about this description. What kind of events occur during delivery?

Reply:
Isn't it just a list of deliveries? Meaning that orders with split packages (different sellers, etc) have multiple delivery events? Would this apply for pnp?